docs/ValidationCriteriaMaster.md

```diff
@@ -0,0 +1,426 @@
+
+> - Given a set of properties taken from the [Properties Table](#Properties-Table)
+> - When the IfcEntity, and optionally the Type, exists
+> - Then the IfcEntity is associated at most to the property set with the PropertySet Name
+</details>
+
+<details><summary>PNAM_01: The property set does not contain unrequested properties</summary>
+
+> - Given a set of properties taken from the [Properties Table](#Properties-Table)
+> - When the IfcEntity, and optionally the Type, exists
+> - And the IfcEntity is associated to a property set with the PropertySet Name
+> - Then the property set has at most the properties with the Property Name
+</details>
+
+<details><summary>PTEX_01: Property values belong to a list of values</summary>
+
+> - Given a set of properties taken from the [Properties Table](#Properties-Table)
+> - When the IfcEntity, and optionally the Type, exists
+> - And the IfcEntity is associated to a property set with the PropertySet Name
+> - And the property set has a property with the Property Name
+> - Then the property value is part of the List Of Values
+</details>
+
+<details><summary>PVAL_01: Property values are not null and not empty</summary>
+
+> - Given a set of properties taken from the [Properties Table](#Properties-Table)
+> - When the IfcEntity, and optionally the Type, exists
+> - And the IfcEntity is associated to a property set with the PropertySet Name
+> - And the property set has a property with the Property Name
+> - Then the property value is not null
+> - And the property value is not empty
+</details>
+
+<details><summary>PVAL_02: Requested property value types are found</summary>
+
+> - Given a set of properties taken from the [Properties Table](#Properties-Table)
+> - When the IfcEntity, and optionally the Type, exists
+> - And the IfcEntity is associated to a property set with the PropertySet Name
+> - And the property set has a property with the Property Name
+> - And the property value is not null
+> - Then the property type is equal to the Property Value Type
+</details>
+
+## Spatial decomposition
+
+| **RULE ID** | **CRITERIA**                      | **VALUE [examples]**               | **ENTITY (if applicable)** | **CT (if applicable)** |
+|-------------|-----------------------------------|------------------------------------|----------------------------|------------------------|
+| SDEC_01     | Spatial decomposition is verified | As per Spatial Decomposition Table | na                         | Spatial Decomposition  |
+
+> **Acceptance criteria**: For the **Spatial decomposition** capability, the validation procedure must verify that a Parent Element of the requested type aggregates (via `IfcRelAggregates`) exactly a given number of Child Elements of the requested type, no more and no less.
+
+**Bullet point example**:
+- IfcRailway *(Name: LO1336)*
+  - IfcFacilityPart.TRACKSTRUCTURE *(Name: LO1336-BC-BC01)*
+  - IfcFacilityPart.TRACKSTRUCTURE *(Name: LO1336-BC-BC02)*
```

Comment:
I find this much more readable than the tables above.

Reply:
Fair enough. Let's leave the decision to the TI author. (Markdown one, as ugly as it is, can be easily converted in CSV, JSON, Excel, and your good old friend....LaTex ;) )